SolWatch/Game1.cs

```diff
@@ -95,6 +96,32 @@ protected override void Draw(GameTime gameTime)
                 effects: SpriteEffects.None,
                 layerDepth: 0f);
 
+            // Draw "North" arrow
+            spriteBatch.Draw(
+                texture: arrowTexture,
+                destinationRectangle: new Rectangle(
+                    location: new Point(100, 180),
+                    size: new Point(80, 80)),
+                sourceRectangle: null,
+                color: Color.White,
+                rotation: 0f,
+                origin: new Vector2(arrowTexture.Width / 2, arrowTexture.Height / 2),
```

Comment:
Don't need to do this now, but could write a `Utilities.TextureCenter(Texture2D)` helper method.